encrypted-config-value-bundle/src/test/resources/bad.key

```diff
@@ -0,0 +1 @@
+AES:OMG
```

Comment:
Is there a difference between a real 256 bit AES key that can't decrypt the secret vs. invalid AES?